server/templates/pages/src/style-guide.html

```diff
@@ -0,0 +1,29 @@
+<!doctype html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="referrer" content="origin">
+        <meta name="robots" content="noindex,nofollow">
+        <link rel="stylesheet" href="/styles/main.css">
+    </head>
+    <body>
+        <label class="mdl-checkbox mdl-js-checkbox" for="checkbox-1">
+          <input type="checkbox" id="checkbox-1" class="checkbox_test">
+          <span class="checkbox_label">Checkbox 1</span>
+        </label>
+        <label class="mdl-checkbox mdl-js-checkbox" for="checkbox-2">
+          <input type="checkbox" id="checkbox-2" class="checkbox_test">
```

Comment:
in the style guide: let's `check` one of the checkboxes and one of the radio buttons by default (on page load) so we can see the styles right away